src/common/polyfill/URL.ts

```diff
@@ -0,0 +1,59 @@
+import { btoa } from "./Base64";
+import { Blob } from "./Blob";
+
+export class URL {
+  /**
+   * fake createObject, use base64 instead
+   * @param blob
+   */
+  static createObjectURL(blob: Blob) {
+    const buffer = blob.buffers[0];
+    const type = typeof blob.type === "object" ? blob.type.type : blob.type;
+    const base64 = _arrayBufferToBase64(buffer);
+    const prefix = `data:${type};base64,`;
+    return prefix + base64;
+  }
+
+  static revokeObjectURL(url: string) {
+    // Do nothing
+  }
```

Comment:
never be called, should delete this function?

Reply:
This is called by `request` and `MeshOptDecoder`